src/core_plugins/kibana/public/home/components/tutorial/string_parameter.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export function StringParameter({ id, label, value, setParameter }) {
+  const handleChange = (evt) => {
+    setParameter(id, evt.target.value);
+  };
+
+  return (
+    <div className="kuiSideBarFormRow">
+      <label
+        className="kuiSideBarFormRow__label"
+      >
+        {label}
+      </label>
+      <div className="kuiSideBarFormRow__control kuiFieldGroupSection--wide">
+        <input
+          className="kuiTextInput"
+          type="number"
```

Comment:
I need to introduce a new entry in `PARAM_TYPES` for string parameters. The Netflow module, for example, will want to provide users with a parameter to override the Elasticsearch hostname:port in the `onPremCloud` scenario. This doesn't play well with the `<input>` element's `type` attribute set to `number` here.

Reply:
string has already been added

Reply:
You're right, there is a string param type. However, what I said about it not playing well with this input field still seems to be the case.